packages/pv-create-component/.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
```

Comment:
should also be in repo-root

Reply:
most of the rules are in the .gitignore file in the root directory but each package might need it's own additional rules (e.g. here for the lib folder)